### PR TITLE
Fixed #853 Made Sidebar collapsible for small screens.

### LINKED ISSUE #853: admin tickets view sidebar on small screens
admin's tickets view for conferences on small devices has sidebar uncollapsible
screen shot 2016-03-09 at 10 14 33 pm

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
#853 

image

image

### CODE REVIEW

**Comment by nikhilgupta1211** [comment]
@Ana06 please can you have a look at this and tell if it is ok for the sidebar to come from the top or it should come from the side?

File: app/views/layouts/_admin.html.haml [hunk 1]
```diff
@@ -1,14 +1,20 @@
 .container-fluid
+  - if @conference and not @conference.short_title.nil?
```
Comment (nikhilgupta1211) [inline app/views/layouts/_admin.html.haml]: Actually i thought of this only but as we are using the condition `if @conference and not @conference.short_title.nil?` for the admin_sidebar  so i used it here also &#128517;. Should i push the changes ?

Comment (differentreality) [inline app/views/layouts/_admin.html.haml]: Please commit the change @ChrisBr asked for!

Comment (nikhilgupta1211) [inline app/views/layouts/_admin.html.haml]: Done 😊

**Comment by nikhilgupta1211** [comment]
@ChrisBr i have made the changes

**Comment by nikhilgupta1211** [comment]
@differentreality can you please have a look at this :)

**Comment by differentreality** [comment]
@ChrisBr is already assigned on this issue, so he will take a look when he can. And frankly I prefer someone with more experience with frontend stuff to review this. This is a bit too much css for my taste.

**Comment by differentreality** [comment]
@nikhilgupta1211 this should be possible using the bootstrap way, without adding the extra CSS. Wanna give it a go?

**Comment by nikhilgupta1211** [comment]
@differentreality sure :), In this for collapse I am using bootstrap and the extra css is only for the animation of cross converting into the bars. Can you please explain a little more that how should I change it ?

**Comment by differentreality** [comment]
I am not sure what you are asking. What does 'cross converting into the bars' mean?
What I am saying is look at the hamburger menu we already have, it is generated automatically by bootstrap without requiring any extra CSS. Why not just do the same for the second hamburger menu?

**Comment by nikhilgupta1211** [comment]
Sorry, actually I meant this
image
Actually I didn't used the bootstrap ham their, because it would have been same as the navbar ham.
Should I change it to the bootstrap ham menu?

**Comment by differentreality** [comment]
Yes, it makes sense to use the tools we already have, rather than introducing custom CSS.

**Comment by nikhilgupta1211** [comment]
Ok, thanks :)

**Comment by nikhilgupta1211** [comment]
image

image

@differentreality I have removed the extra css, please have a look at it :)

**Comment by differentreality** [comment]
Why aren't you removing the rest of the CSS you have added? You can implement the same thing using bootstrap navbar.

**Comment by nikhilgupta1211** [comment]
@differentreality, we are using
```
position: absolute; // for positioning otherwise If I would use it in a navbar-header class of bootstrap its position will not be correct, also it will shift the sidebar down
  z-index: 1000; // as we want the button to be on the navbar otherwise it will not be clickable when the bar is expanded
  right: -10px;
  top: 50px;
  margin-bottom: 10px;

  .icon-bar { // we have defined white color as default for the of navbars in osem so I changed the color here to black so it would be visible
    background-color: black;
  }
```
Also we are not using the .navbar.navbar-default classes for the sidebar.
Sorry, but can you please tell how should I do this with bootstrap?

**Comment by differentreality** [comment]
Why not use navbar classes?

**Comment by nikhilgupta1211** [comment]
@differentreality please have a look at this :)

image

**Pull request merged**